### ISSUE TITLE
Possibility to calculate 4K but make it looks like 1440p?

### ISSUE THREAD

**Issue by lucanello** [avibrazil/RDM#40]
HiDPI scales everything 2x which is needing too many resources on my Macbook when using the scaled resolution 1440p because it is calculated as 5K.

So is there a possibility like on Windows to stick to 4K which is the native resolution of my external monitor and simply scale the UI differently (e.g. 125%)?

I am using macOS Catalina 10.15.4.

**Comment by koraytugay** [comment]
No, that is not how macOS works.. Google "ui scaling windows vs mac" please.
